test/uk/gov/hmrc/transitmovements/services/ObjectStoreServiceSpec.scala

```diff
@@ -110,6 +119,64 @@ class ObjectStoreServiceSpec
       }
     }
 
+    "should add the content to a file" in {
+      val objectSummary: ObjectSummaryWithMd5 = arbitraryObjectSummaryWithMd5.arbitrary.sample.get
+
+      val filePath =
+        Path.Directory(s"movements/${arbitraryMovementId.arbitrary.sample.get}").file(randomUUID.toString).asUri
+
+      val file                          = new java.io.File("test/uk/gov/hmrc/transitmovements/data/valid.xml")
+      val path: java.nio.file.Path      = file.toPath
+      val source: Source[ByteString, _] = Source.single(ByteString("this is test content")) //FileIO.fromPath(path)
+
+      when(
+        mockObjectStoreClient.putObject(
+          any[File],
+          any[Source[ByteString, _]],
+          any[RetentionPeriod],
+          any[Option[String]],
+          any[Option[Md5Hash]],
```

Comment:
We know the retention period (7 years), source (`source`) and MD5 hash (from `objectSummary`), so we should ensure they're being used here by using `eqTo`.